.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -0,0 +1,165 @@
+.pull_test_dockers:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
+  needs: []
+  tags: ["runner:docker"]
+  rules:
+    !reference [ .on_system_probe_changes_or_manual ]
+  stage: kernel_matrix_testing
+  script:
+    # DockerHub login for build to limit rate limit when pulling base images
+    - DOCKER_REGISTRY_LOGIN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.$DOCKER_REGISTRY_LOGIN_SSM_KEY --with-decryption --query "Parameter.Value" --out text)
+    - aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.$DOCKER_REGISTRY_PWD_SSM_KEY --with-decryption --query "Parameter.Value" --out text | docker login --username "$DOCKER_REGISTRY_LOGIN" --password-stdin "$DOCKER_REGISTRY_URL"
+    # Pull base images
+    - mkdir $KITCHEN_DOCKERS
+    - inv -e system-probe.save-test-dockers --output-dir $KITCHEN_DOCKERS --arch $ARCH
+    # upload docker images to S3 artifacts bucket
+    - pushd $DD_AGENT_TESTING_DIR
+    - tar czvf kitchen-dockers-$ARCH.tar.gz kitchen-dockers-$ARCH
+    - $S3_CP_CMD kitchen-dockers-$ARCH.tar.gz $S3_ARTIFACTS_URI/kitchen-dockers-$ARCH.tar.gz
```

Comment:
Is there a reason we are copying to S3 *and* using Gitlab artifacts?

Reply:
Leftover from previous work. Removed!